pkg/workload/histogram/exporter/util.go

```diff
@@ -15,17 +14,9 @@ import (
 )
 
 var (
-	invalidCharRegex      = regexp.MustCompile(`[^a-zA-Z0-9_]`)
-	invalidFirstCharRegex = regexp.MustCompile(`^[^a-zA-Z_]`)
-	summaryQuantiles      = []float64{50, 95, 99, 100}
+	summaryQuantiles = []float64{50, 95, 99, 100}
 )
 
-func sanitizeOpenMetricsLabels(input string) string {
-	sanitized := invalidCharRegex.ReplaceAllString(input, "_")
-	sanitized = invalidFirstCharRegex.ReplaceAllString(sanitized, "_")
-	return sanitized
-}
-
```

Comment:
Removing this in favour of using the `roachprod-microbench/util` to maintain consistency with `roachtest/clusterstats` package